### PR TITLE
Build level interface

### PR DESCRIPTION
**Pull request by GorginZ**
Configuration for LevelOne starting state: starting positions for walls, dots, pacman, monsters.

the interface implementation seems arbitrary but can be reconsidered in the future, for now at least have the level to start working with and implementing other PacMan logic.

### CODE REVIEW

**Review by franklin-ross — APPROVED** [review]
Looks good to me

File: PacMan/CellState.cs [hunk 1]
```diff
@@ -3,9 +3,9 @@ namespace PacMan
   public enum CellState
   {
 
-    Dot = 0,
+    O = 0,
```
Comment (bcarnevale-myob) [inline PacMan/CellState.cs]: What is the reason for the change from 'Dot' to 0? and same for 'Wall'?

Comment (GorginZ) [reply PacMan/CellState.cs]: wanted a easily visible visual confirmation of my hard coded level wall coordinates. No point writing a test for it because there's no logic it's just being set, but I wanted to check when I re-did the numbers after I changed the dimensions that I did move all the walls to the actual places I want them (I did find I had two coordinates wrong).

I forgot to change this back, I should have picked it up in the changes. From now on it will be the 'word' because I don't need to look at it anymore because I know my level matches my plan and I can refer to my plan grid and know it reflects where things 'start'.
